reaver/run.py

```diff
@@ -75,7 +82,10 @@ def main(argv):
     sess_mgr = rvr.utils.tensorflow.SessionManager(sess, expt.path, args.ckpt_freq, training_enabled=not args.test)
 
     env_cls = rvr.envs.GymEnv if '-v' in args.env else rvr.envs.SC2Env
-    env = env_cls(args.env, args.render, max_ep_len=args.max_ep_len)
+    env = env_cls(args.env, args.render,
```

Comment:
This would break support for non-sc2 environments. 
Can you add a general-purpose flag similar to `render` to base env class (e.g. call it `monitor`)? Simply ignore it in other envs for now.